ui/admin/app/components/form/worker/create-worker-led/index.js

```diff
@@ -30,7 +30,7 @@ export default class FormWorkerCreateWorkerLedComponent extends Component {
   @tracked newWorkerValue;
 
   get clusterIDFromURL() {
-    const hostname = this.browserObject.hostname;
```

Comment:
Are the `?` used here for [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)? if so, due to the importance of `window` and `location` we can assure it will exist.

Reply:
@calcaide yea not sure on this one, I took expression from the hostname getter inside of the `browserObject` service.  My guess is they didn't want it to throw an error if it didn't exist, but I can see how that would never be a problem.

Reply:
Yeah, as Zed mention is fine eitherway and this is a not blocking comment (sorry I didn't make that clear at first).

Yeah, this could be a problem on server side rendering but I don't think we are going to reproduce it since we do not perform that.

How my mind see it: if we do not have `window` on the current context of Boundary Admin UI, the browser, then we are in trouble :sweat_smile:

But you guys are right, better to be safe than sorry.